README.md

```diff
@@ -57,38 +57,74 @@ helm delete --purge che-plugin-registry
 docker run -it  --rm  -p 8080:8080 eclipse/che-plugin-registry
 ```
 ### Plugin meta YAML structure:
-Here is an overview of all fields that can be present in plugin meta YAML files
-
-|field name in YAML | description | mandatory |
-|-------|-------|-------|
-| publisher | publisher name | yes |
-| name | plugin name | yes |
-| version | plugin version | yes |
-| deprecate.migrateTo | which plugin should be used instead when plugin is deprecated. Format: publisher/name/version | no |
-| deprecate.autoMigrate | if deprecated plugin should be automatically replaced by another plugin specified in deprecate.migrateTo | no |
-| type | plugin type | yes|
-| displayName | plugin name | yes|
-| title| plugin title | yes |
-| description| plugin description | yes |
-| icon| URL to plugin icon (must be in SVG format) | yes |
-| url| an URL to the plugin source | yes |
-| publisher| name of plugin publisher | yes |
-| repository | URL to repository of the plugin | yes |
-| category| plugin category | yes<sup>1</sup> |
-| firstPublicationDate | date of publishing the plugin (in ISO 8601) | no<sup>2</sup> |
-| latestUpdateDate | date of latest plugin update (in ISO 8601) | no<sup>3</sup> |
-| preview | a URL to devfile, to preview this plugin | no |
-| tags | a list of tags, related to this plugin | no|
-| mediaImage | links for images showcasing the plugin | no |
-| mediaVideo | links for video showcasing the plugin| no |
-| attributes | a map of special attributes, can be used for instance, imported plugins from VS Code | no |
+Here is an overview of all fields that can be present in plugin meta YAML files. This document represents the current `v3` version.
+
+```yaml
+apiVersion:            # v2; v1 supported for backwards compatability
+publisher:             # publisher name
+name:                  # plugin name
+version:               # plugin version
+type:                  # plugin type; e.g. "Theia plugin", "Che Editor"
+displayName:           # name shown in user dashboard
+title:                 # plugin title
+description:           # short description of plugin's purpose
+icon:                  # link to SVG icon
+repository:            # repo hosting plugin sources
+category:              # see [1]
+firstPublicationDate:  # optional; see [2]
+latestUpdateDate:      # optional; see [3]
+deprecate:             # optional; section for deprecating plugins in favor of others
+  autoMigrate:         # boolean
+  migrateTo:           # new plugin id
```

Comment:
Plugin id as in publisher/name/version? What if it's in a different plugin registry?

Reply:
Yes, new plugin ID. It doesn't make sense to support migrating to another registry IMO since it would be something we don't control. It's intended purpose is to allow a smoother transition between plugin ids *within* the registry. 

@l0rd can confirm.